### PR TITLE
fix typos [ci skip]

### PR DESCRIPTION
**Pull request by rajatbansal93**
fix typos in file:
`guides/source/contributing_to_ruby_on_rails.md`

### CODE REVIEW

File: guides/source/contributing_to_ruby_on_rails.md [hunk 1]
```diff
@@ -633,7 +633,7 @@ been updated.
 Sometimes you will be asked to make some changes to the code you have
 already committed. This can include amending existing commits. In this
 case Git will not allow you to push the changes as the pushed branch
-and local branch do not match. Instead of opening a new pull request,
+and local branch does not match. Instead of opening a new pull request,
```
Comment (sikachu) [inline guides/source/contributing_to_ruby_on_rails.md]: I thought _the pushed branch and local branch_ are plural so you have to use _do_.

File: guides/source/contributing_to_ruby_on_rails.md [hunk 2]
```diff
@@ -58,14 +58,14 @@ WARNING: Please do not report security vulnerabilities with public GitHub issue
...
-Later in this guide you'll find detailed instructions for proposing a patch to
+Later in this guide, you'll find detailed instructions for proposing a patch to
 Ruby on Rails. If you enter a wish list item in GitHub Issues with no code, you
 can expect it to be marked "invalid" as soon as it's reviewed.
 
 Sometimes, the line between 'bug' and 'feature' is a hard one to draw.
 Generally, a feature is anything that adds new behavior, while a bug is
 anything that causes incorrect behavior. Sometimes,
-the core team will have to make a judgement call. That said, the distinction
+the core team will have to make a judgment call. That said, the distinction
```
Comment (maclover7) [inline guides/source/contributing_to_ruby_on_rails.md]: `judgement` is the correct spelling 😬

File: guides/source/contributing_to_ruby_on_rails.md (hunk 1 above)
Comment (maclover7) [inline guides/source/contributing_to_ruby_on_rails.md]: 👍 `do` is the correct form here

**Comment by maclover7** [comment]
Not sure why the PR was closed, but manually pulled in via https://github.com/rails/docrails/commit/cf2158cbbf12a3a00c780ef5136f11b77c566713.

**Pull request closed without merging**